lib/Mail.php

```diff
@@ -654,10 +654,14 @@ private function getDebugMailRecipients(array $recipients): array
     private function renderParams(string $string): string
     {
         $templatingEngine = \Pimcore::getContainer()->get('pimcore.templating.engine.delegating');
-        $twig = $templatingEngine->getTwigEnvironment();
+        $twig = $templatingEngine->getTwigEnvironment(true);
         $template = $twig->createTemplate($string);
 
-        return $template->render($this->getParams());
+        $rendered = $template->render($this->getParams());
+
+        $templatingEngine->disableSandboxExtensionFromTwigEnvironment();
+
+        return $rendered;
```

Comment:
What about writing it like this?

```suggestion
        try {
            $twig = $templatingEngine->getTwigEnvironment(true);
            $template = $twig->createTemplate($string);

            return $template->render($this->getParams());
        } finally {
            $templatingEngine->disableSandboxExtensionFromTwigEnvironment();
        }
```

Reply:
yes, looks cleaner. thanks!